### ISSUE TITLE
TSC Meeting Minutes (10th October 2023) 

### ISSUE THREAD

**Issue by nzlosh** [StackStorm/discussions#126]
October 2023 @StackStorm/tsc 1 hour meeting plan:

StackStorm TSC Meetings #33 for the schedule.
TLDR; Jitsi link: https://meet.jit.si/StackStormTSC

Tuesday 10 October 2023

Attendees
=========
- Aj
- Amanda
- Amanda
- Carlos
- Dale Smith @ Orchestral
- Dan Porter
- David Culbreth @ H-E-B
- Dmitri Zimine
- Eugen C.
- FileMagic
- Jacob Floyd
- Marcel Weinberg
- Mick McGrath
- Ravi Rao
- Ronnie Hoffmann
- Ryan
- Scott

Action points
=============

Carlos will create an issue to provide details on what needs to be fixed for St2 CI pipeline intermittent failures so contributers know what needs to be fixed.

Scott can help with providing patches for the v3.8.1 release that will be focused on security patches.

Eugen C. has announced he'll be stepping down as a StackStorm Maintainer after being the release manager for `v3.8.1`

Project Administration / StackStorm security
============================================

https://github.com/orgs/StackStorm/projects/25

Review of project administration is driven by the issues in the project.

 - Migrating api.stackstorm.com is required.
 - How to signing packages/docker images.
 - How to create reproducible builds using Pants.

Supporting other Linux Operating Systems
========================================

Amazon Linux 2 and CentOS9 with newer version of Python.

Next major release of StackStorm will require supporting Python3.8 to 3.11
and should include the support matrix between OS, RabbitMQ, Redis and MongoDB.

A coordination manager is required.

StackStorm Web site
===================

Dmitri and Dale will work on migrating the website to Eleventy.

---

_Thank you everyone who took the time to participate in this months TSC._
